routes/admin.js

```diff
@@ -1,8 +1,42 @@
 const express = require('express')
 const router = express.Router()
+const passport = require('passport')
 
 router.get('/', function(request, response){
-  response.render('admin')
+  request.url < 2 ? response.render('admin') : response.redirect('/loggedout')
```

Comment:
Is this supposed to be request.url.length? I'm a little confused by there being two 
'/' routes. One in admin.js and one in index.js. Shouldn't there only be one ?